src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,304 @@
+package duke;
+
+import duke.fileHandling.FileHandler;
+import duke.task.*;
```

Comment:
As far as possible, it's best to avoid wildcard imports; try to list all imports individually instead